examples/sushi/config.py

```diff
@@ -53,5 +67,33 @@
 
 required_approvers_config = Config(
     default_connection=DuckDBConnectionConfig(),
-    users=[User(username="test", roles=[UserRole.REQUIRED_APPROVER])],
+    users=[
+        User(
+            username="test",
+            roles=[UserRole.REQUIRED_APPROVER],
+            notification_targets=[
+                SlackApiNotificationTarget(
+                    notify_on=["apply_start", "audit_failure"],
```

Comment:
From what I can tell, it seems like if I subscribe to these events then I will get all of them notified to me instead of just those that are relevant to me? For example in prod I want audits to notify owners of the model upon failure but when in dev I want apply and audits to notify the plan author of failure. From what I can tell in the current implementation I will get notified for everything. Is that right?